README.md

```diff
@@ -1,2 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Contact-Form-Training
-This a contact form built during the training exercise for the Code Reviewer role
+
+> This is the second training project for the code reviewer role. It is a simple Contact Form project built with HTML and CSS.
+
+
+## Built With
+
+- HTML
+- CSS
+- Git
+  
+
+## Live Demo (if available)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/1GORDON)
+- Twitter: [@twitterhandle](https://twitter.com/TinyefuzaG)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tinyefuza-gordon/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Hey @1GORDON,  I see you used [this template](https://github.com/microverseinc/readme-template) for your readme file but you forgot to customize it to your project. It would be nice to customize the readme template to suit your project, so that your repository will look more professional and other users can get a clearer description of your project and the author.

Reply:
Thanks for the review. I will fix it immediately